src/rgw/services/svc_meta_be_sobj.cc

```diff
@@ -152,11 +153,22 @@ int RGWSI_MetaBackend_SObj::get_entry(RGWSI_MetaBackend::Context *_ctx,
   string oid;
   ctx->module->get_pool_and_oid(key, &pool, &oid);
 
-  return rgw_get_system_obj(*ctx->obj_ctx, pool, oid, *params.pbl,
-                            objv_tracker, params.pmtime,
-                            y, dpp,
-                            params.pattrs, params.cache_info,
-                            params.refresh_version);
+  int ret = 0;
+  if (get_raw_attrs) {
+    ret = rgw_get_system_obj(*ctx->obj_ctx, pool, oid, *params.pbl,
+                              objv_tracker, params.pmtime,
+                              y, dpp,
+                              params.pattrs, params.cache_info,
+                              params.refresh_version, true);
+  } else {
+    ret = rgw_get_system_obj(*ctx->obj_ctx, pool, oid, *params.pbl,
+                              objv_tracker, params.pmtime,
+                              y, dpp,
+                              params.pattrs, params.cache_info,
+                              params.refresh_version);
```

Comment:
this last argument is `bool raw_attrs`, can we pass `get_raw_attrs` directly there without the if-else?

Reply:
done